docs/adoption.md

```diff
@@ -129,26 +129,32 @@ Finally, in GitHub actions secrets, set `AZUREAPPSERVICE_PUBLISHPROFILE_PRODUCTI
-| `APP_SERVICE_NAME`           | Service name (must be unqiue across whole of Azure) e.g. `housing-repairs-scheduling-api-{LOCAL_AUTHORITY_NAME}`     |
-| `AUTHENTICATION_IDENTIFIER`  | A unique identifier used to validate access used to validate access                                                  |
-| `DRS_API_ADDRESS`            | DRS API address, e.g. `https://yourserver/OTWebServiceGateway_INSTANCENAME/ws/soap?wsdl`                             |
-| `DRS_CONTRACT`               | Contract value to use when making requests to DRS                                                                    |
-| `DRS_LOGIN`                  | DRS login/user name                                                                                                  |
-| `DRS_PASSWORD`               | DRS password                                                                                                         |
-| `DRS_PRIORITY`               | Priority to use when making requests to DRS                                                                          |
-| `JWT_SECRET_PRODUCTION`      | JWT secret generated for for production                                                                              |
-| `JWT_SECRET_STAGING`         | JWT secret generated for for staging                                                                                 |
-| `NUGET_AUTH_GITHUB_TOKEN`    | Authentication token for authenticating with GitHub NuGet feed                                                       |
-| `NUGET_AUTH_GITHUB_USERNAME` | Username for authenticating with GitHub NuGet feed                                                                   |
-| `RESOURCE_GROUP_LOCATION`    | Azure Resource Group location, e.g. `UK South`                                                                       |
-| `RESOURCE_GROUP_NAME`        | Azure Resource Group name                                                                                            |
-| `SENTRY_DSN`                 | [Sentry Data Source Name](https://docs.sentry.io/product/sentry-basics/dsn-explainer/)                               |
-| `SERVICE_PLAN_NAME`          | Service plan name (must be unique across whole of Azure) e.g. `housing-repairs-schduling-api-{LOCAL_AUTHORITY_NAME}` |
-| `STATE_CONTAINER_NAME`       | The name of the container to store Terraform state in                                                                |
-| `STATE_KEY_NAME`             | The file path and name of your Terraform state file                                                                  |
-| `STORAGE_ACCOUNT_NAME`       | Storage account name for Terraform state, e.g. `housing-repairs-online`                                              |
+| Secret name                            | Description                                                                                                          |
```

Comment:
The entire table is showing as changed due to widening the first column to accommodate longer names.
The focus of the changes are the additions/changes to lines 135 - 146.

Removing whitespace difference highlighting _maybe_ helpful.